run/rclone_archive/verify-and-configure-archive.sh

```diff
@@ -15,7 +15,7 @@ function verify_configuration () {
         exit 1
     fi
 
-    if ! rclone lsd "$RCLONE_DRIVE": | grep -q "$RCLONE_PATH"
+    if ! rclone lsd --max-depth 2 "$RCLONE_DRIVE": | grep -q "$RCLONE_PATH"
```

Comment:
https://rclone.org/commands/rclone_lsd/ says that 'lsd' does not recurse by default. Should the `-R` option also be specified, or does `--max-depth 2` enable recursion too?

Reply:
I tried -R and the --max-depth flag didn't change the behavior, which was over a minute going through all nested albums over the years.

Reply:
Not sure I understand your comment. Does the -R flag cause this take over a minute, or the --max-depth flag, or is it always that slow?

Reply:
Sorry for the confusion.

-R overrides the —max-depth flag going to infinite recursion which is slow depending on the drive. —max-depth is faster, but maybe bumping to 3 would allow greater flexibility.

Reply:
Isn't the better solution to use `rclone lsd gphotos:album` instead of `rclone lsd --max-depth=2 gphotos:`?
The former is much faster for me.
In other words, it seems to me that:
```
if ! rclone lsd "$RCLONE_DRIVE": | grep -q "$RCLONE_PATH"
```
should just be:
```
if ! rclone lsd "$RCLONE_DRIVE":"$RCLONE_PATH"
```